common-api/src/main/java/org/pmiops/workbench/exceptions/ExceptionUtils.java

```diff
@@ -0,0 +1,66 @@
+package org.pmiops.workbench.exceptions;
+
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import java.io.IOException;
+import java.net.SocketTimeoutException;
+import javax.servlet.http.HttpServletResponse;
+import org.springframework.http.HttpStatus;
+
+/** Utility methods related to exceptions. */
+public class ExceptionUtils {
```

Comment:
Does DB communicate with firecloud or notebooks? Seems that this class in the workbench is only used in those scenarios.

Reply:
Removed this and changed the way exceptions are caught / thrown.